features/trusted-types.yml

```diff
@@ -3,17 +3,16 @@ description: Trusted types allow you to lock down insecure parts of the DOM API
 spec: https://w3c.github.io/trusted-types/dist/spec/
 caniuse: trusted-types
 usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/2722
-# toJSON() is excluded because they are not needed to use trusted types in the
-# idiomatic way. toJSON() shipped in Chrome 90, and the rest in Chrome 83.
+entrypoint: api.trustedTypes
```

Comment:
@foolip we talked about this on our call today. Instead of doing this, what do you think of something like:

```yaml
status:
  compute_from:
    - api.trustedTypes
```

I'd expect an error to throw when `Set(data.status.compute_from).isSubsetOf(Set(data.compat_features)) === false`